src/main/java/ch/njol/skript/SkriptAddon.java

```diff
@@ -149,40 +103,23 @@ public SkriptAddon setLanguageFileDirectory(String directory) {
 		Language.loadDefault(this);
 		return this;
 	}
-	
+
 	@Nullable
 	public String getLanguageFileDirectory() {
 		return languageFileDirectory;
 	}
-	
+
 	@Nullable
 	private File file = null;
-	
+
 	/**
-	 * @return The jar file of the plugin. The first invocation of this method uses reflection to invoke the protected method {@link JavaPlugin#getFile()} to get the plugin's jar
-	 *         file. The file is then cached and returned upon subsequent calls to this method to reduce usage of reflection.
+	 * @return The jar file of the plugin.
+	 * The first invocation of this method uses reflection to invoke the protected method {@link JavaPlugin#getFile()} to get the plugin's jar file.
+	 * The file is then cached and returned upon subsequent calls to this method to reduce usage of reflection.
```

Comment:
Caching is promised here, yet I don't see any?

Reply:
I believe the file parameter is still there, so this can be easily updated for the cache to work again